packages/eas-cli/src/build/utils/printBuildInfo.ts

```diff
@@ -87,3 +99,13 @@ export function printDeprecationWarnings(deprecationInfo?: DeprecationInfo): voi
     Log.warn(deprecationInfo);
   }
 }
+
+export function printUserError(error: errors.UserError, { printNewLine = true } = {}): void {
+  if (printNewLine) {
+    Log.addNewLineIfNone();
+  }
+  Log.error(error.message);
+  if (error.fyiURL) {
+    Log.error(learnMore(error.fyiURL, { dim: false }));
+  }
+}
```

Comment:
```suggestion
export function printUserError(error: errors.UserError): void {
  Log.error(error.message);
  if (error.fyiURL) {
    Log.error(learnMore(error.fyiURL, { dim: false }));
  }
}
```

You call this function only once with pritnNewLine true, so it's probably cleaner to add `Log.addNewLineIfNone();` there.